utils.js

```diff
@@ -1,4 +1,36 @@
-function getYouTubeVideoID(url) {
+// Function that can be used to wait for a condition before returning
+async function Wait(condition, timeout = 5000, check = 100) { 
```

Comment:
Can you change this to camelCase. `wait()`

Reply:
Its one word is camelCase needed

Reply:
yes

Reply:
Changed